marquez_airflow/models.py

```diff
@@ -12,13 +10,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Usage: $ ./init-marquez-db.sh
-
-set -eu
 
-psql -v ON_ERROR_STOP=1 --username "${POSTGRES_USER}" > /dev/null <<-EOSQL
-  CREATE USER ${MARQUEZ_USER};
-  ALTER USER ${MARQUEZ_USER} WITH PASSWORD '${MARQUEZ_PASSWORD}';
-  CREATE DATABASE ${MARQUEZ_DB};
-  GRANT ALL PRIVILEGES ON DATABASE ${MARQUEZ_DB} TO ${MARQUEZ_USER};
-EOSQL
+class AirflowTaskMeta:
+    def __init__(self, name, source_type=None, source_name=None,
```

Comment:
Are we assuming a single source for all inputs and outputs? I don't think that's the case.
It looks like the original code had a notion of source for a dataset for this purpose: https://github.com/MarquezProject/marquez-airflow/pull/49/files#diff-7e0f7695907de94fcc84e2a2575bdb7bL50

Reply:
A source maps to an [`postgres_conn_id`](https://airflow.apache.org/docs/stable/_modules/airflow/operators/postgres_operator.html) in Airflow operator. You can provide an override for the `database` that's then used by [`PostgresHook`](https://airflow.apache.org/docs/stable/_modules/airflow/hooks/postgres_hook.html). But, it'd still be a single source that can be set to override what's in the connection. Really, we'd need to support parsing queries that happen across databases. I opened the following issues as follow up work: #50 , #51